### ISSUE TITLE
ToT & Focus Frame: Can't Remove Buff/Debuff Icons or Timers

### ISSUE THREAD

**Issue by RopeDrink** [Wutname1/SpartanUI#84]
Greetings.

Short and sweet: the target of target bar & focus bar will still show buff/debuff timers and icons even when you have buffs/debuffs disabled for said bars. This issue has been ongoing for a few years now - I managed to remedy it in the past by butchering the LUA's somehow but I can't seem to find/remember how. Seeing as they can't be removed entirely, the best I can do is set the icons to size 1, but you can still see the pixel representing them and mass-buffs results in maximum-size overlapped timers regardless of what you do.

Screenshot:
https://imgur.com/jPYNTtf

In addition, resorting to LUA butchery means the fix is reset each and every time the addon gets updated (and it'll class itself as out-of-date if edited).

I could live with this if I could remember what I did specifically to the LUA in order to entirely remove all icons & timers from the target/target & focus bar, but back then I recall it taking me forever to pinpoint the exact lines to do this without annihilating the entire addon.

Any feedback would be welcome.

**Comment by RopeDrink** [comment]
Hello,

I have had buffs/debuffs and all relevant settings disabled for those frames for the guts of a year, yet this problem has been persistent through many patches and updates to SUI. My only solution has been to manually enter the LUA files and 'move' the X & Y off-set of the buffs/debuffs on TargetOfTarget and Focus Frames so that those buffs/debuffs go off-screen, so even then they are not truly disabled.

I've disabled them using the regular options, disabled them via the LUA, even set the sizes to 0 in the hopes they'd disappear but even that never worked, in that order - yet no matter what I did, past or present, those timers and icons remained. Reducing their size would make the icons tiny but stack the timers (which always remained same size regardless of the icon size) and thus stack over eachother, making it even more frustrating.

**Comment by RopeDrink** [comment]
I might backup my current UI and try a completely clean install, but again, this problem has been prevalent for a few expansions for me, so I have a feeling that may not work.

Addons I'm using:

AnyFont
Bartender4 (integral)
DeadlyBossMods (mostly disabled)
DejaCharacterStats
Doom_CooldownPulse
gmFonts
LoggerHead
MoveAnything
Raven (What I used for buffs/debuffs, hence wanting to remove those from Spartan)
Recount
RSA
ScrollingCombatText
SharedMediaAdditionalFonts
SpartanUI

Last I checked, this problem persisted with or without Raven installed - the only mod that would effect buffs or debuffs. Everything else is for fonts or non-frames.
